docs/DeveloperGuide.md

```diff
@@ -894,28 +888,40 @@ testers are expected to do more *exploratory* testing.
-Planned enhancements would be to ensure that, when unassigning weddings, case is ignored and the Wedding::isSameWedding(Wedding) function is used
-to check for same-ness.
-5. The `untag PERSON_INDEX t/TAG_NAME` command is case-sensitive for the TAG_NAME. So, only exactly matching the case of the
-tag as stored in WedLinker will untag it from the Person. This limits the speed with which users can use the application and does not
-follow the case sensitivity defined for Tag in the Tag::isSameTag(Tag) function or the intended real-world case-sensitivity.
-Planned enhancements would be to ensure that, when untagging Person objects, case is ignored and the Tag::isSameTag(Tag) function is used
-to check for same-ness.
+
+4. **Make `unassign-wedding` case-insensitive**: The `unassign-wedding PERSON_INDEX w/WEDDING_NAME` command is case-sensitive for the `WEDDING_NAME`. This means that the command will only unassign a Wedding if the case of the `WEDDING_NAME` exactly matches case of the word as stored in WedLinker.
+This limits the speed with which users can use the application, and does not 
+align with the case sensitivity defined for Weddings in the `Wedding::isSameWedding(Wedding)` function, nor does it reflect real-world case-sensitivity of wedding names.
+The planned enhancement would be to ensure that when unassigning weddings, case is ignored and the `Wedding::isSameWedding(Wedding)` function is used
+to check for ensure proper matching of Wedding regardless of case.
+
+5. **Make `untag` case-insensitive**: The `untag PERSON_INDEX t/TAG_NAME` command is case-sensitive for the `TAG_NAME`.
+This means that the command will only untag a Tag if the case of the `TAG_NAME` exactly matches case of the word as stored in WedLinker.
+This limits the speed with which users can use the application and does not
+align with the case sensitivity defined for Tag in the `Tag::isSameTag(Tag)` function, nor does it reflect real-world case-sensitivity of tag names.
+The planned enhancement would be to ensure that when untagging Person objects, case is ignored and the `Tag::isSameTag(Tag)` function is used
+to check for ensure proper matching of Tag regardless of case.
+
+6. **Allow searching contacts with blank fields**: Currently, the `find` command does not support searching for contacts with blank field values.
+For example, using the command `find a/`, where the address field is empty, will display an error on the GUI telling the user that the field to be searched cannot be empty. 
+However, contacts can currently be added with all fields, other from name, blank. Thus, there may be valid cases where users want to search for contacts with certain blank fields. 
+The planned enhancement would be to modify the `find` command to allow users to search for contacts with blank fields.
+When no keywords are provided after a prefix (e.g., `find a/`), the application should return a list of contacts where the corresponding field is blank.
+
```

Comment:
Sorry to have not mentioned this earlier, but another planned enhancement I believe is adding processing for multiple partner tags in the assign-wedding instruction that considers the order of the p1/ and p2/ tags or rejects the command if there are multiple partner tags specified for one wedding.

Also, I think UI auto-updating after certain commands (I think unassign-wedding, for example) might be a planned enhancement? I'm actually not sure if this qualifies as planned enhancement